src/edit/actions/DragAction.js

```diff
@@ -12,8 +12,8 @@ L.DragAction = L.EditAction.extend({
     L.EditAction.prototype.initialize.call(this, map, overlay, options);
   },
 
-  addHooks: function() {
-    const edit = this._overlay.editing;
+  addHooks() {
+    let edit = this._overlay.editing;
```

Comment:
Hi, here, i think we can use a const, and it'll be OK! But the above change is great, on line 2. Thank you!